src/components/tabs/TabGroup.svelte

```diff
@@ -1,10 +1,13 @@
 <script>
   export let active = 0;
   import { writable } from "svelte/store";
-  import { setContext } from "svelte";
+  import { createEventDispatcher, setContext } from "svelte";
 
   const store = writable(active);
   setContext("activeTab", store);
+
+  const dispatch = createEventDispatcher();
+  store.subscribe((value) => dispatch("tabChanged", { active: value }));
```

Comment:
This took a while for me to understand. If the goal is to dispatch `tabChanged` when the value of `active` changes, I think we could make `active` a store and set a watcher to dispatch `tabChanged`. If you prefer to keep `store`, let's change it to a more descriptive name rather than a generic `store`.
(unrelated but not sure why the default value of active is 0, isn't it supposed to be a string?)